static/app/components/replays/diff/replaySliderDiff.tsx

```diff
@@ -63,6 +66,23 @@ function DiffSides({leftOffsetMs, replay, rightOffsetMs, viewDimensions, width})
     },
   });
 
+  const organization = useOrganization();
+  const [dividerClicked, setDividerClicked] = useState(false); // never flips back to false
```

Comment:
I would create this with `useRef` instead, right now it'll cause a re-render which isn't needed as we're just tracking the value not rendering anything.

```
  const dividerClickedRef = useRef(false);
  ...
  if (organization && !dividerClickedRef.current) {
    ...
    dividerClickedRef.current = true
```